daemon/internal/newrelic/integration/test.go

```diff
@@ -285,10 +289,15 @@ func ScrubHost(in []byte) []byte {
 	return re.ReplaceAll(in, []byte("__HOST__"))
 }
 
-func SubEnvVars(in []byte) []byte {
+func (t *Test) subEnvVars(in []byte) []byte {
 	re := regexp.MustCompile("ENV\\[.*?\\]")
 	return re.ReplaceAllFunc(in, func(match []byte) []byte {
-		return []byte(os.Getenv(string(match[4 : len(match)-1])))
+		k := string(match[4 : len(match)-1])
+		v, present := os.LookupEnv(k)
```

Comment:
This appears to give the OS environment values precedence over `t.Env` when substitution occurs?  Is this desired or is it better to let the `/*ENVIRONMENT*/` values be able to override the OS environment?

Reply:
@mfulb Very good point! I think values from `/*ENVIRONMENT*/` directive should take precedence over OS environment. How about https://github.com/newrelic/newrelic-php-agent/commit/5064b7c93267fae793fbef33760b1df37fc35d77?